.github/workflows/tests.yml

```
@@ -0,0 +1,36 @@
+name: Open Source Programs Run Tests
+
+on: [push, pull_request]
+
```

```suggestion
branches: [ develop ]
```
Can you add branches also?

---

I don't think it is necessary as it should run on all branches of this repository

---

Okay, I didn't quite sure about this thanks for clearnig :)